### DISCUSSION TITLE
SolidJS / SolidStart / Vite support

### DISCUSSION THREAD

**Discussion by AshSimmonds** [iway1/trpc-panel#34] [General]
Perhaps it already does, just with different syntax, but I'm not smart enough to figure it out.

**Comment by AshSimmonds** [comment]
Trying to add it here, feel free to PR:

https://github.com/AshSimmonds/solidstart-17

**Comment by iway1** [comment]
Thanks! 

In theory it should work pretty easily w/ any backend since it just renders to a text string and I assumed all backend frameworks support text responses.

Not Vite support b/c it would require importing the router into front end code? I'd thought people would want to avoid doing that but not 100% sure
